citi-onboarding/virtu-onepage
# Feature/ourhistory

**What I did:**
- Firts thing i did was position the elements in their places 
- Then i started the compotentization
- Styled the elements and made the section responsive 

**How to test:**
- Go to the client folder and start the react server with *npm start* or *yarn start*
- To see the responsivity resize your screen 

resolves #19
